CONTRIBUTING.md

```diff
@@ -1,62 +1,69 @@
 <h2 align="center">Contributing</h2>
 
-<p align="center">Contributing to Dark Reader can be in a variety of ways. Here, you can learn more about it. Thank you in advance for reading this <code>CONTRIBUTING.md</code>.</p>
+<p align="center">YOU can contribute to Dark Reader in a variety of ways!  Here, you can learn more.  Thank you in advance for reading this document.</p>
 
 ## Sponsor!
 
-<a href="https://opencollective.com/darkreader/donate" target="_blank" rel="noreferrer noopener"> <img src="https://opencollective.com/darkreader/donate/button@2x.png?color=blue" width=300 /></a>
+Please sponsor the development of Dark Reader by clicking this link:  
 
-Sponser the development of Dark Reader
+<a href="https://opencollective.com/darkreader/donate" target="_blank" rel="noreferrer noopener"> <img src="https://opencollective.com/darkreader/donate/button@2x.png?color=blue" width=300 /></a>
 
-## Translation
+## Translate!
 
-[Improve or suggest](https://github.com/darkreader/darkreader/tree/master/src/_locales) a translation.
+[Improve or suggest](https://github.com/darkreader/darkreader/tree/master/src/_locales) a translation.  
 See the list of [language codes](https://developer.chrome.com/webstore/i18n#localeTable) that we can support.
 
-## Fix a wrong inversion!
+## Fix a Wrong Inversion!
 
-If a website is **already dark**, that regardless of system's preferred color-scheme has all pages dark by default. You can **add it to the [dark-sites.config](https://github.com/darkreader/darkreader/blob/master/src/config/dark-sites.config) file**
-*(please, preserve the alphabetical order)*.
+If a website is **already dark** (meaning that regardless of the system's preferred color-scheme, has all pages dark by default), you can **add it to the [dark-sites.config](https://github.com/darkreader/darkreader/blob/master/src/config/dark-sites.config) file** *(please, preserve alphabetical order)*.
```

Comment:
"already dark" shouldn't mean/equal that regardless [...].

Let's try something along the lines of `If a website is **by-default** darkened ([...])`
That makes more sense to me.